.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,6 +10,19 @@ Fixes #
 
 **Special notes for your reviewer**:
 
+**Checklist**
+
+This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
+Approvers are expected to review this list.
+
+- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
+- [ ] PR: The description will help future contributors
```

Comment:
```suggestion
- [ ] PR description: The description will help future contributors
```

Reply:
I kept PR:, but enhanced the text, thanks!